pages/customer/[id]/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useContext } from "react"
```

Comment:
Could you add links to these detail pages on the list that the user finds on `/customer`. Clicking on a customer on the list should open the corresponding detail page.